src/main/kotlin/no/skatteetaten/aurora/boober/service/RedeployService.kt

```diff
@@ -24,55 +28,82 @@ class RedeployService(val openShiftClient: OpenShiftClient, val openShiftObjectG
         }
     }
 
-    fun triggerRedeploy(deploymentSpec: AuroraDeploymentSpec, redeployContext: RedeployContext): RedeployResult {
+    fun triggerRedeploy(deploymentSpec: AuroraDeploymentSpec, imageStream: ImageStream?, deploymentConfig: DeploymentConfig?): RedeployResult {
         val type = deploymentSpec.type
         if (type == TemplateType.build || type == TemplateType.development) {
-            return RedeployResult(message = "No deploy was made with $type type")
+            return RedeployResult(message = "No deploy was made with $deploymentSpec.type type")
```

Comment:
Her tror jeg du må ha { } rundt ${deploymentSpec.type} for at det skal bli rett.